tests/subsys/logging/dictionary/pytest/test_logging_dictionary.py

```diff
@@ -43,7 +43,12 @@ def process_logs(dut: DeviceAdapter, build_dir):
     # timeout earlier with per-test timeout.
     handler_output = dut.readlines_until(regex = '.*##ZLOGV1##[0-9]+', timeout = 600.0)
 
-    encoded_logs = handler_output[-1]
+    # Join all the output lines together
+    handler_output = ''.join(ho.strip() for ho in handler_output)
```

Comment:
Not sure `ho` was the best choice for a variable name 😆

Reply:
Err.... hm.... hm... Santa! Think Santa Clause! :D

Reply:
Santa wouldn't strip